src/_format.ts

```diff
@@ -84,8 +84,9 @@ export function getFormat(
   indent: string | number | undefined;
   whitespace: { start: string; end: string };
 } {
+  const defaultIndent = opts?.indent ?? 2;
   if (!obj || typeof obj !== "object" || !(ftmSymbol in obj)) {
-    return { indent: opts?.indent, whitespace: { start: "", end: "" } };
+    return { indent: defaultIndent, whitespace: { start: "", end: "" } };
```

Comment:
refactor nitpick: since this value is either user option or default `2`, it is not default, we could inline it (also makes computation only when we are in the `if` branch

Reply:
by default, i more meant "base" (like the indent level before any detection happens)

ill inline it